examples/with_http_get/handler.js

```diff
@@ -0,0 +1,42 @@
+import * as https from "https";
+import { pathToFileURL } from "url";
+
+// GET request sample
+export async function handle(event, context) {
+  let dataString = "";
+
+  const response = await new Promise((resolve, reject) => {
+    const req = https.get(
+      "https://pokeapi.co/api/v2/pokemon/ditto",
+      function (res) {
+        res.on("data", chunk => {
+          dataString += chunk;
+        });
+
+        res.on("end", () => {
+          resolve({
+            statusCode: 200,
+            headers: { "Content-Type": ["application/json"] },
+            body: JSON.stringify(JSON.parse(dataString), null, 4),
+          });
+        });
+      },
+    );
```

Comment:
I don't think you need to fix it, but IMO mixing callbacks and promises is confusing.

Reply:
Yes I agree, but this code was already here so out of scope for this PR probably